### PR TITLE
Added ability to cancel multiple other effects to effects.

### PR DESCRIPTION
**Pull request by kevingranade**
The only down side is we lose the messages that confirm curing various things, but I feel like that can go in the effect being cured, thoughts?

### CODE REVIEW

File: data/json/effects.json [hunk 1]
```diff
@@ -851,6 +861,7 @@
         "rating": "good",
         "max_intensity": 2,
         "int_dur_factor": 50,
+	"removes_effects": ["infected", "bite", "bleed", "fungus", "dermatic"],
```
Comment (Rivet-the-Zombie) [inline data/json/effects.json]: sp: dermatik

**Comment by KA101** [comment]
Might be a problem if we need some way to differentiate cures by source, but ATM I don't think that's an issue.

File: data/json/effects.json [hunk 2]
```diff
@@ -829,6 +829,16 @@
     },
     {
         "type": "effect_type",
+        "id": "cureall",
```
Comment (KA101) [inline data/json/effects.json]: "//" : "Add new royal-jelly-fixable effects here", perhaps.

**Comment by kevingranade** [comment]
Two nice catches, thanks.

**Pull request merged**